locale/en/security.md

```diff
@@ -51,8 +51,8 @@ release process above to ensure that the disclosure is handled in a consistent m
 
 Security notifications will be distributed via the following methods.
 
-- [http://groups.google.com/group/nodejs-sec](http://groups.google.com/group/nodejs-sec)
-- [http://blog.nodejs.org](http://blog.nodejs.org)
+- [https://groups.google.com/group/nodejs-sec](https://groups.google.com/group/nodejs-sec)
+- [https://blog.nodejs.org](https://blog.nodejs.org)
```

Comment:
Can you please update the link to: https://nodejs.org/en/blog/
Also, we have an RSS feed for this as well: https://nodejs.org/en/feed/vulnerability.xml